utils/durable/activities/index.ts

```diff
@@ -55,15 +55,20 @@ export const createActivity = <
 ): Promise<ActivityResult<F | S>> => {
   const logger = createLogger(context, activityName);
 
-  return fromEither(InputCodec.decode(rawInput))
-    .mapLeft(errs =>
-      failActivity(logger)(
-        "Error decoding activity input",
-        readableReport(errs)
-      )
-    )
-    .chain(input => body({ context, input, logger }))
-    .map(e => OutputCodec.encode(e))
-    .fold<ActivityResult<F | S>>(identity, identity)
-    .run();
+  return pipe(
+    rawInput,
+    InputCodec.decode,
+    TE.fromEither,
+    x => x,
```

Comment:
why do we need this?

Reply:
uh, of course it's just a test that I forgot to remove, thanks!

Reply:
removed in [8907d1a](https://github.com/pagopa/io-functions-pushnotifications/pull/36/commits/8907d1a8c21784a1d4d9097a279c8e094a4d8c7c)